ui/oopsyraidsy/data/05-shb/trial/emerald_weapon-ex.js

```diff
@@ -0,0 +1,113 @@
+import NetRegexes from '../../../../../resources/netregexes.js';
+import ZoneId from '../../../../../resources/zone_id.js';
+
+export default {
+  zoneId: ZoneId.CastrumMarinumExtreme,
+  damageWarn: {
+    'EmeraldEx Heat Ray': '5BD3', // Emerald Beam initial conal
+    'EmeraldEx Photon Laser 1': '557B', // Emerald Beam inside circle
+    'EmeraldEx Photon Laser 2': '557D', // Emerald Beam outside circle
+    'EmeraldEx Heat Ray 1': '557A', // Emerald Beam rotating pulsing laser
+    'EmeraldEx Heat Ray 2': '5579', // Emerald Beam rotating pulsing laser
+    'EmeraldEx Explosion': '5596', // Magitek Mine explosion
+    'EmeraldEx Tertius Terminus Est Initial': '55CD', // sword initial puddles
+    'EmeraldEx Tertius Terminus Est Explosion': '55CE', // sword explosions
+    'EmeraldEx Airborne Explosion': '55BD', // exaflare
+    'EmeraldEx Sidescathe 1': '55D4', // left/right cleave
+    'EmeraldEx Sidescathe 2': '55D5', // left/right cleave
+    'EmeraldEx Shots Fired': '55B7', // rank and file soldiers
+    'EmeraldEx Secundus Terminus Est': '55CB', // dropped + and x headmarkers
+    'EmeraldEx Expire': '55D1', // ground aoe on boss "get out"
+    'EmeraldEx Aire Tam Storm': '55D0', // expanding red and black ground aoe
+  },
+  shareWarn: {
+    'EmeraldEx Divide Et Impera': '55D9', // non-tank protean spread
+    'EmeraldEx Primus Terminus Est 1': '55C4', // knockback arrow
+    'EmeraldEx Primus Terminus Est 2': '55C5', // knockback arrow
+    'EmeraldEx Primus Terminus Est 3': '55C6', // knockback arrow
+    'EmeraldEx Primus Terminus Est 4': '55C7', // knockback arrow
+  },
+  triggers: [
+    {
+      // Orbs from Aetheroplasm Production must be taken with a friend.
+      id: 'EmeraldEx Aetheroplasm',
+      // 55AE = Nitrosphere Aetheroplasm
+      // 55AF = Ceruleum Sphere Aetheroplasm
+      damageRegex: ['55AE', '55AF'],
+      condition: (e) => e.type === '15',
+      mistake: (e, data, matches) => {
+        return {
+          type: 'fail',
+          blame: matches.target,
+          text: {
+            en: `${matches.ability} (alone)`,
+            fr: `${matches.ability} (seul(e))`,
```

Comment:
```suggestion
            fr: `${matches.ability} (seul(e))`,
            ko: `${matches.ability} (혼자 맞음)`,
```

Reply:
Sorry, I've removed this from the PR because I haven't been back in emerald and I don't think these are working.  I think in general it's better to wait until after PRs have landed to add translations.  I should have said the same to Akurosia before merging in (and then removing) those translations.